modules/apps/style-book/style-book-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -1,179 +1,209 @@
-@import 'atlas-variables';
+@import 'cadmin-variables';
 
 $controlMenuDesktopHeight: 56px;
 $previewInfoBarHeight: 36px;
 $toolbarDesktopHeight: 56px;
 
-.style-book-editor {
-	--style-book-color-picker-color: none;
+html#{$cadmin-selector} {
+	.cadmin {
```

Comment:
Hey @bryceosterhaus and @pat270 maybe we could bring this to the bundler? in the npm-scripts builder we add a selector wrapper since this will only be used for DXP modules, also I don't know if it's possible but it would be great because we would ensure that all modules that use npm-scripts are under the scope of `cadmin` and developers don't have to worry about it and it would lessen the effort to change each product. What do you think?

Reply:
Hmm, I'm not sure we could do this accurately within npm-scripts because often the root of peoples modules is a `*.jsp` file. If we were isolating this to just react components, that is possible with the bundler, but I don't know about being able to scope all modules with this class. Also, I don't think we want every module that uses npm-scripts to be in the scope of cadmin, I may be wrong on that though...

> npm-scripts builder we add a selector wrapper since this will only be used for DXP modules

Are we already doing this somewhere? I am not familiar with this.

Reply:
Hmm yeah, makes sense. We're not actually using 😁, just an idea that came to mind, I'm not sure this would also be possible would have to be investigated but even if it was, I agree with you that it's hard to track where we should add scope.